tests/unit/s2n_tls13_server_cert_test.c

```diff
@@ -167,6 +168,38 @@ int main(int argc, char **argv)
         /* free memory allocated in s2n_alloc*/
         free(tls13_cert.data);
     }
+
+    /* Test server sends cert and client receives cert for tls 1.3 */
```

Comment:
Just to make sure I'm interpreting this correctly, is this just a backfill test for a unit test that was missing or is this related to client auth/client certificate request validation?

Reply:
Added sanity check to enusre this isn't the reason client auth isn't working